src/data/error.rs

```diff
@@ -287,6 +324,45 @@ pub enum LexError {
     #[error("no newline at end of file")]
     NoNewlineAtEOF,
 
+    #[error("unknown token: {0:?}")]
+    UnknownToken(u8),
```

Comment:
Can you print this as a `char` instead? I know this isn't unicode safe but the current output isn't very helpful:

```
`
<stdin>:1:1 error: invalid token: unknown token 96
```

Reply:
What should I do when it's not unicode valid? Panicing might come back to bite us.

Reply:
Maybe try converting to Unicode and print as hex char literal if it can't.

Reply:
This already panics: https://github.com/jyn514/rcc/issues/265. Casting to `char` can't panic anyway since it will turn it into the appropriate unicode scalar value.